oembed/v1.0/oembed.md

```diff
@@ -0,0 +1,64 @@
+# ClickView oEmbed API
+
+[oEmbed](https://oembed.com/) is an open format designed to allow embedding content from a website into another page.
+
+The ClickView oEmbed API can be used to obtain the the necessary information to embed ClickView videos into your website.
+
+
+## Resource URL
+
+```http
+GET https://online.clickview.com.au/oembed
+GET https://online.clickview.co.uk/oembed
+GET https://online.clickview.co.nz/oembed
+```
```

Comment:
Once fixed, we should ask them to retrieve the oembed link from our discoverability tags instead of hard coding them, so that we retain control over what that endpoint looks like. This also will make dealing with our clickv.ie/w links much easier for them as they won't have to have a map to figure out which oembed endpoint to use.

Reply:
I shall update the documentation once this is live. Until then this is still correct.